src/tests/multihost/ad/test_ad_misc.py

```diff
@@ -323,3 +323,33 @@ def test_0004_bz2110091(multihost, adjoin, create_aduser_group):
         log2 = re.compile(r'\[recreate_ares_channel.*Initializing.new.c-ares.channel', re.IGNORECASE)
         assert log1.search(dom_log), 'Destroying the old c-ares related log missing'
         assert log2.search(dom_log), 'Initializing new c-ares related log missing'
+
+    @staticmethod
+    def test_0005_bz2133854(multihost, adjoin, create_aduser_1kadgrp):
+        """
+        :title: sdap_add_incomplete_groups called with ignore_group_members enabled
+        :id: 3fcdacb1-7985-4f6e-a381-b65e83b5e516
+        :customerscenario: true
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2133854
+        :setup:
+          1. Join RHEL-client to AD
```

Comment:
For upstream this could be just a client right?

Reply:
corrected